docs/public/specifications/tests/DNSSEC-TP/dnssec10.md

```diff
@@ -76,13 +76,13 @@ DS10_HAS_NSEC3                     | INFO    |            | Consistent NSEC3 ret
 DS10_INCONSISTENT_NSEC_NSEC3       | ERROR   |2 ns_ip_list| Some servers return NSEC, others return NSEC3.
 DS10_MISSING_NSEC_NSEC3            | ERROR   | ns_ip_list | Missing expected NSEC or NSEC3 in a signed zone.
 DS10_MIXED_NSEC_NSEC3              | ERROR   | ns_ip_list | Both NSEC and NSEC3 are returned from the same server.
-DS10_NAME_NOT_COVERED_BY_NSEC      | ERROR   | ns_ip_list | The non-existent name is not correctly covered by the NSEC records.
-DS10_NAME_NOT_COVERED_BY_NSEC3     | ERROR   | ns_ip_list | The non-existent name is not correctly covered by the NSEC3 records.
-DS10_NON_EXISTENT_RESPONSE_ERROR   | ERROR   | ns_ip_list | No or error in response of an expected non-existent name.
+DS10_NAME_NOT_COVERED_BY_NSEC      | ERROR   | ns_ip_list | The nonexistent name is not correctly covered by the NSEC records.
+DS10_NAME_NOT_COVERED_BY_NSEC3     | ERROR   | ns_ip_list | The nonexistent name is not correctly covered by the NSEC3 records.
+DS10_NON_EXISTENT_RESPONSE_ERROR   | ERROR   | ns_ip_list | No or error in response of an expected nonexistent name.
```

Comment:
The messages in Zonemaster::Engine are different, but do erroneously spell `non-existent` as `nonexistent`.

Reply:
I can drop, but my preference is to move both repositories to nonexistent. I can also move this to a different PR...

Reply:
Wikipedia has `nonexistent` main entry, but `non-existent` as an alternative.

* https://en.wiktionary.org/wiki/nonexistent#English
* https://en.wiktionary.org/wiki/non-existent#English

Reply:
However, dictionary.com has no entry for “non-existent”; only “nonexistent”. I suggest we go ahead fixing the spelling mistake both here and in the msgid.

I was going to say that there is a discrepancy between the spec and the code in the messages, but the specification of DNSSEC10 merely states “Description of when message tag is outputted”. However, more recent specifications do have an exact copy of the msgids in that column. Shouldn’t we create an issue about that?

Reply:
I do not think we need to create an issue because we know that we have old specifications that need an update (and @tolvmannen has started with the first in that list, Address01).

Reply:
@marc-vanderwal, I think it would be wrong to consider "non-existent" to be an error just because dictionary.com is lacking it. My printed New World Dictionary states that "a hyphen may be used after _non-_". Both alternatives are as correct.